### PR TITLE
[DSM] PEPPER-280

### PR DESCRIPTION
**Pull request by pegahtah**
applying the renames and also changing the names of the columns
Screen Shot 2022-11-17 at 3 57 34 PM
Screen Shot 2022-11-17 at 3 57 57 PM
Screen Shot 2022-11-17 at 3 59 41 PM
Screen Shot 2022-11-17 at 4 00 04 PM

### CODE REVIEW

**Review by GiCharkviani — DISMISSED** [review]
LGTM

**Review by SimoneMaiwald — CHANGES_REQUESTED** [review]
I don't think all these parameters where changed

File: ddp-workspace/projects/ddp-dsm-ui/src/app/filter-column/models/column.model.ts [hunk 1]
```diff
@@ -89,7 +89,7 @@ export class ParticipantColumn {
   public static TUMOR_TYPE = new ParticipantColumn('Tumor Type', 'tumorType', 't');
   public static H_E = new ParticipantColumn('H&E', 'hE', 't');
   public static PATHOLOGY_REPORT = new ParticipantColumn('Pathology Report', 'pathologyReport', 't');
-  public static COLLABORATOR_SAMPLE_ID = new ParticipantColumn('Collaborator Sample ID', 'collaboratorSampleId', 't');
+  public static COLLABORATOR_SAMPLE_ID = new ParticipantColumn('Tumor Collaborator Sample ID', 'collaboratorSampleId', 't');
```
Comment (SimoneMaiwald) [inline ddp-workspace/projects/ddp-dsm-ui/src/app/filter-column/models/column.model.ts:92]: Can you also change that label on the tissue page then

File: ddp-workspace/projects/ddp-dsm-ui/src/app/filter-column/models/column.model.ts [hunk 2]
```diff
@@ -118,7 +118,8 @@ export class ParticipantColumn {
   public static QC = new ParticipantColumn('QC', 'qc');
 
   // kit
-  public static COLLABORATOR_SAMPLE = new ParticipantColumn('Collaborator Sample ID', 'bspCollaboratorSampleId', 'k');
+  public static COLLABORATOR_SAMPLE = new ParticipantColumn('Normal Collaborator Sample ID', 'bspCollaboratorSampleId', 'k');
```
Comment (SimoneMaiwald) [inline ddp-workspace/projects/ddp-dsm-ui/src/app/filter-column/models/column.model.ts:121]: same here on the sample tab

File: ddp-workspace/projects/ddp-dsm-ui/src/app/filter-column/models/column.model.ts (hunk 1 above)
Comment (pegahtah) [reply ddp-workspace/projects/ddp-dsm-ui/src/app/filter-column/models/column.model.ts:92]: Oh yes good catch!

File: ddp-workspace/projects/ddp-dsm-ui/src/app/tissue/tissue.component.html [hunk 3]
```diff
@@ -171,7 +171,7 @@
     </tr>
     <tr>
       <td class="TD--Padding">Pathology Report</td>
-      <td>
+      <td colspan="3">
```
Comment (pegahtah) [inline ddp-workspace/projects/ddp-dsm-ui/src/app/tissue/tissue.component.html:174]: I moved the colspan to here to fix a display bug for when tissue was not type block not to change how the tissue page looks like (introduced by another PR of mine, PEPPER-285 )

**Review by SimoneMaiwald — APPROVED** [review]

**Review by ngigliotosBroad — APPROVED** [review]

**Pull request merged**